XXX-url.md

```diff
@@ -0,0 +1,170 @@
+| Title  | Implement WHATWG URL Spec   |
+|--------|-----------------------------|
+| Author | @jasnell                    |
+| Status | DRAFT                       |
+| Date   | 2016-06-30T09:00:00-07:00   |
+
+## Description
+
+The WHATWG URL Standard specifies updated syntax, parsing and serialization of
+URLs as currently implemented by the main Web Browsers. The existing Node.js
+`url` module parsing and serialization implementation currently does not support
+the URL standard and fails to pass 160 of the WHATWG URL parsing tests.
+
+This proposal is to implement the WHATWG URL Standard by modifying the existing
+`url` module to provide an implementation of the `URL` object and associated
+APIs. Doing so improves the robustness of URL parsing, provides consistency
+with browser js code, and can eventually allow the reduction of node.js specific
+APIs.
+
+Initially, the implementation would be introduced as an undocumented
+experimental feature exposed via a new `URL` property in the `url` module.
+
+Because the existing `require('url')` module remains, there should be no
+backwards compatibility concerns. Once a decision is made to provide official,
+documented support for the new URL implementation, a decision would be made
+about whether to switch the internal uses of `require('url').parse()` to the
+new `URL` implementation.
```

Comment:
How about we revisit our wording, because "deprecate" continues to bring grief. So how about APIs that are deprecated mean that we suggest not using it. Possibly also that it gets lower priority with patches.  Then something like "removable" to mean that it's on the roadmap to be removed and there's little sense in attempting to improve it.